spot_ros2_control/CMakeLists.txt

```
@@ -0,0 +1,105 @@
+cmake_minimum_required(VERSION 3.8)
```

copyright

Also

```suggestion
cmake_minimum_required(VERSION 3.22)
```

to match the spot_driver package